src/Vibrato/LFO.cpp

```diff
@@ -0,0 +1,35 @@
+#include <math.h>
+
+#include "LFO.h"
+
+LFO::LFO(float sampleRate, int resolution) : sampleRate(sampleRate), wavetable(resolution) {
+    for (int i = 0; i < resolution; i++) {
+        wavetable.putPostInc(sinf(2 * M_PI * i / resolution));
```

Comment:
I like the way you populated your wavetable. I did mostly the same procedure but with a few more local variables and code lines. This looks to get the same job done more concisely.